### DISCUSSION TITLE
QUESTION MARK IN FLOW/S AND EXPORTER BUT HEALTHY DOCKER SERVICES

### DISCUSSION THREAD

**Discussion by emmanmarquez** [akvorado/akvorado#1029] [Q&A]
Do you have ideas on this?

![image](https://github.com/akvorado/akvorado/assets/123536938/688778ee-d678-4e3a-a287-915337116362)

**Comment by emmanmarquez** [comment]
here's my docker compose ps 

![image](https://github.com/akvorado/akvorado/assets/123536938/c3a54e99-c092-4436-990d-d984ce71c049)

**Comment by vincentbernat** [comment]
There is a whole "troubleshooting" section in the documentation.
